openshift/templates/email-verification-service/email-verification-service-deploy.yaml

```diff
@@ -27,11 +73,13 @@ objects:
   metadata:
     name: ${NAME}
     labels:
-      name: ${NAME}
-      app: ${APP_GROUP}
+      app: ${APP_NAME}${SUFFIX}
+      name: ${NAME}${SUFFIX}
       app-group: ${APP_GROUP}
+      role: ${ROLE}
+      env: ${TAG_NAME}
   spec:
-    host: ${APPLICATION_DOMAIN}
+    host: ${APPLICATION_SERVICE_DOMAIN}
     port:
       targetPort: ${NAME}
```

Comment:
Example of where not applying the ${SUFFIX} consistently could cause connectivity issues.